release/README.md

```diff
@@ -2,253 +2,31 @@
-        $ git push --tags origin
-
-        $ git clone https://github.com/istio/mixer
-        $ cd mixer
-        $ git tag -a ${RELEASE_TAG} -m "Istio Release ${RELEASE_TAG}" ${MIXER_SHA}
-        $ git push --tags origin
-
-        $ git clone https://github.com/istio/auth
-        $ cd auth
-        $ git tag -a ${RELEASE_TAG} -m "Istio Release ${RELEASE_TAG}" ${AUTH_SHA}
-        $ git push --tags origin
-
-### Rebuild artifacts to include the tags
-
-Go to Mixer [stable artifacts](https://testing.istio.io/view/All%20Jobs/job/mixer/job/stable-artifacts/)
-job and click on ```Build with Parameters```.
-Replace ```BRANCH_SPEC``` with the value of ```${RELEASE_TAG}```
-
-Go to Pilot [stable artifacts](https://testing.istio.io/view/All%20Jobs/job/pilot/job/stable-artifacts/)
-job and click on ```Build with Parameters```.
-Replace ```BRANCH_SPEC``` with the value of ```${RELEASE_TAG}```
-
-Go to Auth [stable artifacts](https://testing.istio.io/view/All%20Jobs/job/auth/job/stable-artifacts/)
-job and click on ```Build with Parameters```.
-Replace ```BRANCH_SPEC``` with the value of ```${RELEASE_TAG}```
-
-### Updating ```istio.VERSION```
-
-Now we need update the tags ```istio.VERSION``` to point to the release tag.
-
-        $ git checkout -b ${USER}-${RELEASE_TAG} origin/${BRANCH}
-        $ install/updateVersion.sh -p docker.io/istio,${RELEASE_TAG} \
-           -c docker.io/istio,${RELEASE_TAG} -x docker.io/istio,${RELEASE_TAG} \
-           -i https://storage.googleapis.com/istio-artifacts/pilot/${RELEASE_TAG}/artifacts/istioctl
-
-Create a commit with name "Istio Release ${RELEASE_TAG}", and a PR.
-Once tests are completed, merge the PR, and create an annotated tags
-
-        $ git pull origin ${BRANCH}
-        $ git tag -a ${RELEASE_TAG} -m "Istio Release ${RELEASE_TAG}" HEAD # assuming nothing else was committed
-        $ git push --tags origin
-
-### Creating archives
-
-Sync your workspace at ${RELEASE_TAG}:
-
-        $ git reset --hard ${RELEASE_TAG}
-        $ git clean -xdf
-
-Create the release archives
-
-        $ ./release/create_release_archives.sh
-        # On a Mac
-        $ CP=gcp TAR=gtar ./release/create_release_archives.sh
-        ...
-        Archives are available in /tmp/istio.version.A59u/archives
-
-
-Open the [GitHub Release page](https://github.com/istio/istio/releases),
-and edit the release that points to ```${RELEASE_TAG}```. Uploads the artifacts created by the previous script.
-
-
-### Finalizing the release
-
-Create a PR, where you increment ```istio.RELEASE``` for the next
-release and you update ```istio/downloadIstio.sh``` to point to ```${RELEASE_TAG}```
+When a severe bug is found, developers can contact hklai@google.com and vickyxu@google.com to create a patch release on all
+[currently supported releases]
+(https://docs.google.com/document/d/1scC_f7nObwBut_xxE6lUfvcHBybmttUsJDMwXD3gcO4/edit#heading=h.b56ea7aggz9t). Only bugs that
```

Comment:
Perhaps use the short URL https://goo.gl/dcSBxF   .  When I tried visting the full version with the #heading=... I got an error from google docs.

Reply:
#heading won't work using the shorten link.

Could you try again? It seems to work for me just now.